kubeluigi/k8s.py

```diff
@@ -161,20 +168,23 @@ def print_pod_logs(job: V1Job, pod: V1PodSpec):
     while True:
         try:
             if is_pod_running(pod):
-                logger.info(logs_prefix + " POD is running")
+                logger.debug(logs_prefix + " POD is running")
                 break
             else:
-                logger.info(logs_prefix + " ...waiting for POD to start")
+                logger.debug(logs_prefix + " ...waiting for POD to start")
                 sleep(DEFAULT_POLL_INTERVAL)
         except ApiException as e:
             logger.warning("error while fetching pod logs :" + logs_prefix)
             logger.exception(e)
             raise e
 
-    stream = get_pod_log_stream(pod)
-    for i in stream:
-        l = logs_prefix + ": " + i
-        logger.info(l)
+    try:
+        stream = get_pod_log_stream(pod)
+        for i in stream:
+            l = logs_prefix + ": " + i
+            logger.debug(l)
+    except urllib3.exceptions.ProtocolError:
+        logger.debug("Failed to get pod log stream...")
```

Comment:
maybe good to make this info, otherwise you are left thinking "mm is it running or not?"

Reply:
This exception happens every time we run, so it is not very informative it just adds more noise to the logs.